elements/rh-tile/test/rh-tile.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { expect, html } from '@open-wc/testing';
+import { createFixture } from '@patternfly/pfe-tools/test/create-fixture.js';
+import { sendMouse, sendKeys } from '@web/test-runner-commands';
+import { RhTile } from '@rhds/elements/rh-tile/rh-tile.js';
+
+describe('<rh-tile>', function() {
+  let element: RhTile;
+
+  async function click(element: HTMLElement) {
+    const { x, y, width, height } = element.getBoundingClientRect();
+    const position = [x + width / 2, y + height / 2].map(Math.round) as [number, number];
+    await sendMouse({ type: 'click', position });
+  }
+
+  async function enter() {
+    await sendKeys({ press: 'Enter' });
+    await element.updateComplete;
+  }
+
+  function input(element, checked = false) {
+    return element?.shadowRoot?.querySelector(`input[type="checkbox"]${checked ? ':checked' : ''}`);
```

Comment:
instead of asserting on the shadow root, use a11yShapshot and assert against the ax tree

Reply:
and i think you can inline this